### PR TITLE
Changelogs update

### PR DESCRIPTION
**Pull request by ShivamSh3**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

**Review by sachin-apa — DISMISSED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,41 @@ Dell OpenManage Ansible Modules Release Notes
...
+- idrac_diagnostics - Issue(285322) - This module doesn't support export of diagnostics file to HTTP and HTTPS share via SOCKS proxy.
+- idrac_firmware - Issue(279282) - This module does not support firmware update using HTTP, HTTPS, and FTP shares with authentication on iDRAC8.
+- idrac_storage_volume - Issue(290766) - The module will report success instead of showing failure for new virtual creation on the BOSS-N1 controller if a virtual disk is already present on the same controller.
+- ome_diagnostics - Issue(279193) - Export of SupportAssist collection logs to the share location fails on OME version 4.0.0.
+- ome_smart_fabric_uplink - Issue(186024) - The module supported by OpenManage Enterprise Modular, however it does not allow the creation of multiple uplinks of the same name. If an uplink is created using the same name as an existing uplink, then the existing uplink is modified.
+
+New Modules
+-----------
+
+- dellemc.openmanage.idrac_reset - Resets the iDRAC to factory default settings.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]: Remove this

**Review by rajshekarp87 — CHANGES_REQUESTED** [review]

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -1805,3 +1805,36 @@ releases:
         name: idrac_storage_volume
         namespace: ''
     release_date: '2024-03-29'
+  9.2.0:
+    changes:
+      release_summary: '- idrac_session module is added to allows the creation and deletion of sessions on iDRAC.
+
+        - idrac_reset is enhanced to reset the iDRAC to factory default settings.'
```
Comment (rajshekarp87) [inline changelogs/changelog.yaml]:
```suggestion
        - idrac_reset module is enhanced to support the reset operation'
```

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -1805,3 +1805,36 @@ releases:
         name: idrac_storage_volume
         namespace: ''
     release_date: '2024-03-29'
+  9.2.0:
+    changes:
+      release_summary: '- idrac_session module is added to allows the creation and deletion of sessions on iDRAC.
```
Comment (rajshekarp87) [inline changelogs/changelog.yaml]:
```suggestion
      release_summary: '- idrac_session module is added to manage iDRAC sessions.
```

File: changelogs/changelog.yaml [hunk 4]
```diff
@@ -1805,3 +1805,36 @@ releases:
...
+
+        - idrac_reset is enhanced to reset the iDRAC to factory default settings.'
+      major_changes:
+        - idrac_session - This module is intoduced to allow the creation and deletion of sessions on iDRAC.
+      minor_changes:
+        - idrac_reset - This module is enhanced to reset the iDRAC to factory default settings.
```
Comment (rajshekarp87) [inline changelogs/changelog.yaml]:
```suggestion
        - idrac_reset - This module is enhanced to support the reset operation.
```

**Review by rajshekarp87 — CHANGES_REQUESTED** [review]

**Review by sachin-apa — APPROVED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Review by singh-lovepreet1 — APPROVED** [review]

**Pull request merged**